visualization/plots.py

```diff
@@ -0,0 +1,143 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pickle
+from pathlib import Path
+
+
+def compress_from_file(path):
```

Comment:
You might want to add __init__.py to visualization directory now, this way you will be able to use functions from this file in colab.